cmake/googletest/CMakeLists.txt

```diff
@@ -31,30 +37,28 @@ if(BUILD_TEST)
     endif()
 
     if(WIN32)
-        set(ENABLE_STATIC_DEBUG "-Dgtest_force_shared_crt=ON")
-    endif(WIN32)
-
-    if(DEFINED CMAKE_TOOLCHAIN_FILE)
-        set(TOOLCHAIN_FILE "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
-        set(GTest_DIR ${CMAKE_BINARY_DIR}/dependencies/install/lib/cmake/GTest)
-        set(GTest_DIR ${GTest_DIR} CACHE PATH "" FORCE)
+        list(APPEND GTEST_BUILD_ARGS "-Dgtest_force_shared_crt=ON")
```

Comment:
Please add a `set(GTEST_BUILD_ARGS)` before the if statement like with `set(EXTRA_CMAKE_ARGS)` on line 43.